addons/functions/functions/fnc_placeTrench.sqf

```diff
@@ -76,15 +76,16 @@ ace_trenches_digPFH = [{
     // Stick the trench to the ground
     _basePos set [2, getTerrainHeightASL _basePos];
     private _minzoffset = 0;
-    private ["_ix","_iy"];
+    private _ix = 0;
+    private _iy = 0;
```

Comment:
sry… otherwise sqflint feels unable to be sure about the variable type :grimacing: